src/ol/style/regularshape.js

```diff
@@ -140,6 +146,31 @@ ol.inherits(ol.style.RegularShape, ol.style.Image);
 
 
 /**
+ * Clones the style.
+ * @return {ol.style.RegularShape} The cloned style.
+ * @api
+ */
+ol.style.RegularShape.prototype.clone = function() {
+  var style = new ol.style.RegularShape({
+    fill: this.getFill() ? this.getFill().clone() : undefined,
+    points: this.getPoints(),
+    radius: this.getRadius(),
+    radius2: this.getRadius2(),
```

Comment:
According to the API docs there is also a `radius1` configuration for `RegularShape`. I cannot find it in the OpenLayers code right now, though… :astonished: